### ISSUE TITLE
Note and benefits contradict each other in 1.4.1 (Use of color)

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1032]
Note in [SC 1.4.1](https://w3c.github.io/wcag/understanding/use-of-color.html):

> This success criterion addresses color perception specifically. Other forms of perception are covered in Guideline 1.3 including programmatic access to color and other visual presentation coding. 

Benefits

> People using Braille displays or other tactile interfaces can detect text cues by touch
> People using text-only, limited color, or monochrome displays may be unable to access color-dependent information. 

I understand the note to mean that SC 1.4.1 checks whether colour-coded content is distinguishable without assistive technology. That is why there is also a requirement that the contrast ratio must be at least 3:1 so that sighted users can distinguish different colours (F81, G183). On the other hand, SC 1.3.1 checks whether colour-coded content is transmitted programmatically.

The chapter "Benefits" contradicts this because it suddenly mentions users of assistive technology who cannot see or who do not use the standard view. For these users, however, even a minimum contrast of color-coded content would not be useful. I would therefore like to delete the corresponding users. Does anyone have any objections? 

See my related PRs https://github.com/w3c/wcag/pull/1020 and https://github.com/w3c/wcag/pull/1033

**Comment by patrickhlauke** [comment]
Agree, 1.4.1 has nothing to do with AT users.

**Comment by alastc** [comment]
With #1500 merged, closing this issue.
